nifi-testharness/src/test/java/org/apache/nifi/testharness/samples/Constants.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package org.apache.nifi.testharness.samples;
+
+import java.io.File;
+
+public final class Constants {
+
+    static final File OUTPUT_DIR = new File("./NiFiTest/NiFiReadTest");
+    static final File NIFI_ZIP_DIR = new File("../../nifi-assembly/target");
```

Comment:
No, these are in the test cases directory: they are there as a *sample* (referred to in the documentation), and will not be packaged to the actual API artifact. 
The paths there refer the NiFi bundle produced during the NiFi build, which seems to be a good default in case you are working on the test harness itself.